<launch>
	<rosparam file="$(find explorer_robot_hardware)/config/explorer_arm_controller.yaml" command="load" />
	<!--node name="explorer_arm_controller_spawner" pkg="controller_manager" type="spawner" respawn="false" output="screen" args="explorer_arm_controller" /-->
	<rosparam file="$(find explorer_robot_hardware)/config/explorer_vice_wheel_controller.yaml" command="load" />
	<!--node name="explorer_vice_wheel_controller_spawner" pkg="controller_manager" type="spawner" respawn="false" output="screen" args="explorer_vice_wheel_controller" /-->
	<rosparam file="$(find explorer_robot_hardware)/config/explorer_drive_controller.yaml" command="load" />
	<node name="explorer_drive_controller_spawner" pkg="controller_manager" type="spawner" respawn="false" output="screen" args="explorer_drive_controller" />
	<rosparam file="$(find explorer_robot_hardware)/config/explorer_joint_state_controller.yaml" command="load" />
	<!--node name="explorer_joint_state_controller_spawner" pkg="controller_manager" type="spawner" respawn="false" output="screen" args="explorer_joint_state_controller" /-->
	<node name="explorer_robot_hardware" pkg="explorer_robot_hardware" type="explorer_hardware" output="screen">
		<rosparam file="$(find explorer_robot_hardware)/config/explorer_hardware_cofig.yaml" />
	</node>
</launch>
